lib/iris/analysis/cartography.py

```diff
@@ -668,12 +668,12 @@ def project(cube, target_proj, nx=None, ny=None):
     new_cube = iris.cube.Cube(new_data)
 
     # Add new grid coords
-    x_coord = iris.coords.DimCoord(
-        target_x[0, :], 'projection_x_coordinate',
-        coord_system=copy.copy(target_cs))
-    y_coord = iris.coords.DimCoord(
-        target_y[:, 0], 'projection_y_coordinate',
-        coord_system=copy.copy(target_cs))
+    x_coord = iris.coords.DimCoord(target_x[0, :], 'projection_x_coordinate',
+                                   units='m',
```

Comment:
Define a metre. They are called metres by cartographers, but they aren't *physical* metres in the SI sense...